samples/accessors.cpp

```diff
@@ -75,15 +73,14 @@ int main() {
 
   } catch (exception const& e) {
     std::cout << "SYCL exception caught: " << e.what() << '\n';
-    return 2;
+    return 0;
```

Comment:
Appreciate the discussion around this, but to keep the test failing when there is some error it would be good if the sample could rethrow the exception